pkg/logs/metrics/metrics.go

```diff
@@ -0,0 +1,38 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package metrics
+
+import (
+	"expvar"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/status"
+)
+
+var (
+	logsExpvars    *expvar.Map
+	agentWarnings  = expvar.String{}
+	agentIsRunning = expvar.Int{}
+	// LogsProcessed is the total number of processed logs.
+	LogsProcessed = expvar.Int{}
```

Comment:
What happens when LogsProcessed overflow ?

Reply:
It's an Int64: https://golang.org/pkg/expvar/#Int
If it overflows, then we are rich 💵